homework/src/app.js

```diff
@@ -1,11 +1,157 @@
-{
-  /* Your homework goes here */
-  
-  /* STRINGS */
+ /* STRINGS */
+ let myString = "hello,this,is,a,difficult,to,read,sentence";
+//Add the string to your file and log it.
+console.log(myString);
+
+//Log the length of myString.
+console.log(myString.length);
+
+//The commas make that the sentence is quite hard to read. Find a way to remove the commas from the string and replace them with spaces.
+var newString = myString.replace(/,/g, ' ');
+console.log(newString);
+
+
+/*The string doesn't make any sense. Find a way to reverse this string so that it becomes readable.*/
+
+const str = 'dlroW olleH';
+
+function ReverseString(str) { 
+    return str.split('').reverse().join('') 
+ } 
+ console.log(ReverseString('dlroW olleH')) 
 
   /* ARRAYS */
 
-  /* OPERATORS */
+  let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
```

Comment:
Try and use `const` for variables which are not going to be reassigned 🙂

Reply:
const favoriteAnimals = ["blowfish", "capricorn", "giraffe"];